src/egress.rs

```diff
@@ -11,23 +11,7 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Encode, Decode, TypeInfo, Debug)]
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-#[scale_info(skip_type_params(SnapshotAccLimit, WithdrawalLimit, AssetsLimit))]
-pub enum EgressMessages<
-    AccountId: Ord,
-    Balance: Zero + Clone,
-    ProxyLimit: Get<u32>,
-    WithdrawalLimit: Get<u32>,
-    AssetsLimit: Get<u32>,
-> {
-    EnclaveAccountDump(
-        EnclaveAccountInfoDump<AccountId, Balance, ProxyLimit>,
-        Signature
-    ),
-    EnclaveSnapshot(
```

Comment:
We have consumers of this variant [here](https://github.com/Polkadex-Substrate/orderbook/blob/master/enclave/src/import_blocks.rs#L112) and [here](https://github.com/Polkadex-Substrate/orderbook/blob/master/relayer/src/import.rs#L334). I assume this should be refactored as well?